examples/index_robust04.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+PISA_PREFIX=~/src/pisa
+PISA_BIN=$PISA_PREFIX/build/bin
+THREADS=20
+
+COLLECTION=(/research/remote/collections/TREC/disk4/fr.gz
+	    /research/remote/collections/TREC/disk4/ft.gz
+	    /research/remote/collections/TREC/disk5/fbis.gz
+	    /research/remote/collections/TREC/disk5/latimes.gz)
+
+ROB_PREFIX=./robust04
+
+# Generate robust04 robust04.terms robust04.urls robust04.documents
+zcat ${COLLECTION[@]} | $PISA_BIN/parse_collection -j $THREADS -f trectext --content-parser html --stemmer krovetz -o $ROB_PREFIX
+
+# Generate robust04.docs robust04.freqs robust04.sizes
+$PISA_BIN/invert -i $ROB_PREFIX -o $ROB_PREFIX -j 40 --term-count $(wc -l < $ROB_PREFIX.terms)
+
+# Generate robust04.index.opt
+$PISA_BIN/create_freq_index -t opt -c $ROB_PREFIX -o $ROB_PREFIX.index.opt --check
```

Comment:
`opt` is outdated. We now use `pefopt` (https://github.com/pisa-engine/pisa/pull/192)